googlechrome-90.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": "90.0.4430.72",
+    "description": "Fast, secure, and free web browser, built for the modern web.",
+    "homepage": "https://www.google.com/chrome/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.google.com/chrome/privacy/eula_text.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://dl.google.com/release2/chrome/VYWR7z03VITSenbdO3GEAQ_90.0.4430.72/90.0.4430.72_chrome_installer.exe#/dl.7z",
+            "hash": "704a58c4ccb49d82eea8c6c8354621c9c481fdb0bbc3a070c3d22697475a6332"
+        },
+        "32bit": {
+            "url": "https://dl.google.com/release2/chrome/QXfLrKD3khekJ96AJ562-g_90.0.4430.72/90.0.4430.72_chrome_installer.exe#/dl.7z",
+            "hash": "d29a4c3b994a1f14fec3a67bb44789fe6d44fb0dcc7b7a4a0239f2fa54abadc5"
+        }
+    },
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
```

Comment:
How come this needs an installer step, when the chrome driver and edge driver manifests just said "download the zip, and the binary name is X"?

Reply:
This is all just boilerplate from the original version. It's also in all the other versions: https://github.com/tinymce/scoop-bucket/blob/master/googlechrome-81.json#L20

Also don't forget this is Google Chrome itself, not a webdriver.